src/lib/Metadata/Finalizers/Javadoc/Function.hpp

```diff
@@ -60,9 +60,11 @@ isCopyOrMoveConstructorOrAssignment(FunctionInfo const& I)
     auto const& paramRefType = get<RefType const&>(paramType);
     auto const& paramRefPointee = paramRefType.PointeeType;
     MRDOCS_CHECK_OR(paramRefPointee, false);
-    auto const& paramRefPointeeNamed = get<NamedTypeInfo const&>(paramRefPointee);
-    MRDOCS_CHECK_OR(paramRefPointeeNamed.Name, false);
-    auto const& paramName = paramRefPointeeNamed.Name;
+    auto const* paramRefPointeeNamed = get<NamedTypeInfo const*>(paramRefPointee);
+    if (!paramRefPointeeNamed)
```

Comment:
For reference, this could be `MRDOCS_CHECK_OR(paramRefPointeeNamed, false)`.

MRDOCS_CHECK is that pattern people use with `std::expected` (like `?` in Rust), so that code based on error types doesn't become impossible to read.